Add WG_INTERFACE environment variable that defaults to "pia"

lucasrangit opened the pull request:
Allows overriding the default Wireguard interface "pia" with "wg0" for example.

Useful when you want to match the Wireguard default `/etc/wireguard/wg0.conf` (see https://www.wireguard.com/quickstart/).

Tested on Ubuntu Server 20.04 and https://github.com/linuxserver/docker-wireguard Docker container.

lucasrangit commented:
I didn't see other `WG_` variables documented in the README.md. However, if required I will add this and others.

g00nix commented:
Can you please add quotes based on shellcheck recommendations?

```
148: > "/etc/wireguard/${WG_INTERFACE}.conf" || exit 1
157: wg-quick up "$WG_INTERFACE"
```

I noticed these when reading through the commit. I didn't have time to modify/test myself.

lucasrangit commented:
> Can you please add quotes based on shellcheck recommendations?
> 
> ```
> 148: > "/etc/wireguard/${WG_INTERFACE}.conf" || exit 1
> 157: wg-quick up "$WG_INTERFACE"
> ```
> 
> I noticed these when reading through the commit. I didn't have time to modify/test myself.

Thanks @g00nix for the feedback. I fixed my changes and rebased the PR with the latest master.
